protocol/storage/miner_test.go

```diff
@@ -128,10 +144,26 @@ func TestReceiveStorageProposal(t *testing.T) {
 		assert := assert.New(t)
 		require := require.New(t)
 
-		_, miner, proposal := newMinerTestSetup()
+		porcelainAPI, miner, proposal := newMinerTestSetup()
 		proposal.Payment.Vouchers[0].Signature = types.Signature([]byte{})
 
-		res, err := miner.receiveStorageProposal(context.Background(), proposal)
+		// since we are modifying the deal after we create it, we must resign it
```

Comment:
Ditto, refactor the testing helper methods to allow this test to construct the input it needs more directly.

If that's too onerous, at least break out this signing logic to a new helper so the test code itself is direct.